mavros_extras/msg/DistanceSensor.msg

```diff
@@ -0,0 +1,14 @@
+# DISTANCE_SENSOR message data
+
+Header header
+
+sensor_msgs/Range range
+
+uint8 type			# see MAV_DISTANCE_SENSOR enum
+uint8 MAV_DISTANCE_SENSOR_LASER = 0
+uint8 MAV_DISTANCE_SENSOR_ULTRASOUND = 1
+
+uint8 id 			# XXX FIXME enum ??
+
+uint8 orientation 	# rad
```

Comment:
Rad? I don't think so. Uint8 is more likely orientation enum used in params for ahrs.